internals/models.py

```diff
@@ -910,7 +923,10 @@ def crbug_number(self):
 
   def new_crbug_url(self):
     url = 'https://bugs.chromium.org/p/chromium/issues/entry'
-    params = ['components=' + self.blink_components[0] or BlinkComponent.DEFAULT_COMPONENT]
+    if len(self.blink_components) > 0:
+      params = ['components=' + self.blink_components[0]]
+    else:
+      params = ['components=' + BlinkComponent.DEFAULT_COMPONENT]
```

Comment:
If `self.blink_components` is an empty list, the line show `Index Out of Range` Error. Hence, I have changed it to this syntax. 

UPDATE: We may not need to check for the empty list case as I have figured out a way to assign default value to `Feature.blink_components` using the constructor. See  [this](https://github.com/GoogleChrome/chromium-dashboard/pull/1352/commits/d67262c4b566b7cce2f6235ddb069ccb005fd57c) commit.

Reply:
It's probably best to keep this here.  While your linked commit should prevent the error, it's still safer to use defensive coding here.

What you have here is fine, or you could try using `and` and `or` for a more compact expression:
params = ['components=' + (self.blink_components and blink_components[0] or  BlinkComponent.DEFAULT_COMPONENT)]